symengine/ruby/callgrind_test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+valgrind --tool=callgrind --dump-instr=yes --simulate-cache=yes --collect-jumps=yes ruby -S rspec spec
```

Comment:
Can you document how it is used? I am sure there is a solution.

Sent from my mobile phone.
On Aug 8, 2015 8:58 AM, "Abinash Meher" <notifications@github.com> wrote:

> In symengine/ruby/callgrind_test.sh
> <https://github.com/sympy/symengine/pull/487#discussion_r36578955>:
>
> > @@ -0,0 +1,3 @@
> > +#!/bin/bash
> > +
> > +valgrind --tool=callgrind --dump-instr=yes --simulate-cache=yes --collect-jumps=yes ruby -S rspec spec
>
> This was a substitute to automate tasks by rake. Shifting it to bin at
> the root will require writing a path name everytime I want to execute it.
> How about we shift it to the root of the gem dir. I am trying to make it
> run from other place, a LoadError shows up everytime. Or we can bring
> back rake, I know how to do it with rake.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sympy/symengine/pull/487/files#r36578955>.
>